src/telemetry.ts

```diff
@@ -276,23 +276,6 @@ export interface IEventNamePropertyMapping {
             | 'failedToGetCustomEnvVariables';
     };
     [EventName.HASHED_PACKAGE_PERF]: never | undefined;
-    /**
-     * Telemetry event sent after fetching the OS version
-     */
-    [EventName.PLATFORM_INFO]: {
```

Comment:
Unused anyways, but I'd assume this is all covered by vscode common properties already.